solidity/random-beacon/contracts/libraries/DKG.sol

```diff
@@ -394,9 +394,12 @@ library DKG {
         self.sortitionPool.unlock();
     }
 
-    /// @notice Approves DKG result. Can be called after challenge period for the
-    ///         submitted result is finished. Considers the submitted result as
-    ///         valid and completes the group creation.
+    /// @notice Approves DKG result. Can be called after challenge period for
+    ///         the submitted result is finished. For the first
+    ///         `resultSubmissionEligibilityDelay` blocks after the end of the
+    ///         challenge period can be called only by the DKG result submitter.
+    ///         After that can be called by anyone. Considers the submitted
+    ///         result as valid and completes the group creation.
```

Comment:
Just like for its `RandomBeacon` contract counterpart, let's reorganize documentation to first focus on the effect of calling this function and then on authorization rules:

> Approves DKG result. Can be called when the challenge period for the submitted result is finished. Considers the submitted result as valid, pays reward to the approver, bans misbehaved group members from the sortition pool rewards, and completes the group creation by activating the candidate group. For the first `resultSubmissionEligibilityDelay` blocks after the end of the challenge period can be called only by the DKG result submitter. After that time, can be called by anyone.

Reply:
Done in https://github.com/keep-network/keep-core/pull/2709/commits/2c0a29290112e95b5921f6456cb4adb3e98846d6.